easybuild/easyblocks/g/gcc.py

```diff
@@ -287,6 +294,16 @@ def configure_step(self):
             self.log.info("Performing regular GCC build...")
             configopts += " --prefix=%(p)s --with-local-prefix=%(p)s" % {'p': self.installdir}
 
+        # prioritize lib over lib{64,32,x32} for all architectures by overriding default MULTILIB_OSDIRNAMES config
+        # only do this when multilib is not enabled
+        if self.cfg['prefer_lib_subdir'] and not self.cfg['multilib']:
+            cfgfile = 'gcc/config/i386/t-linux64'
+            multilib_osdirnames = "MULTILIB_OSDIRNAMES = m64=../lib:../lib64 m32=../lib:../lib32 mx32=../lib:../libx32"
+            self.log.info("Patching MULTILIB_OSDIRNAMES in %s with '%s'", cfgfile, multilib_osdirnames)
+            write_file(cfgfile, multilib_osdirnames, append=True)
```

Comment:
@sebth it looks like this is causing problems with older GCC versions, i.e. GCC 4.7.2 and older (except for GCC 4.6.4)... GCC 4.7.3 and newer are OK.

This is the error:
```
/usr/lib/crti.o: could not read symbols: File in wrong format
collect2: ld returned 1 exit status
```

with:

```
$ file /usr/lib/crti.o
/usr/lib/crti.o: ELF 32-bit LSB relocatable, Intel 80386, version 1 (SYSV), not stripped
```

I'll add a `prefer_lib_subdir = False` in those easyconfigs, but do you happen to know what the problem is?